fog/ingest/report/src/lib.rs

```diff
@@ -23,6 +28,48 @@ impl IngestAttestationEvidenceVerifier {
     /// the report data bytes. The details of this are tied to the layout of
     /// the "identity" object in the ingest enclave impl.
     pub fn validate_ingest_attestation_evidence(
+        &self,
+        attestation_evidence: &AttestationEvidence,
+    ) -> Result<RistrettoPublic, Error> {
+        match attestation_evidence {
+            AttestationEvidence::VerificationReport(report) => {
+                self.validate_verification_report(report)
+            }
+            AttestationEvidence::DcapEvidence(evidence) => {
+                let evidence = evidence
+                    .try_into()
+                    .map_err(|_| Error::Encoding(EncodingError::InvalidInput))?;
+                self.validate_dcap_evidence(evidence)
+            }
```

Comment:
I haven't fully traced it down, but I think we'll need to keep both of these implementations around for clients to know how to talk to EPID and DCAP, prior to the DCAP release